libs/coin-modules/coin-cardano/src/synchronisation.unit.test.ts

```diff
@@ -126,31 +109,102 @@ describe("makeGetAccountShape", () => {
 
   describe("spendableBalance", () => {
     it("should return 0 spendable balance when there is no utxos", async () => {
-      const getTransactionsMock = jest.mocked(getTransactions);
       getTransactionsMock.mockReturnValue(
         Promise.resolve({
           transactions: [],
           externalCredentials: [
+            { path: { index: 0 } as BipPath, networkId: "id", isUsed: false, key: "" },
+          ],
+          internalCredentials: [],
+        } as any),
+      );
+      const result = await shape(accountShapeInfo, { paginationConfig: {} });
+      expect(result.spendableBalance).toEqual(new BigNumber(0));
+    });
+
+    it("should return spendable balance with rewards when delegated to dRep", async () => {
+      getTransactionsMock.mockReturnValue(
+        Promise.resolve({
+          transactions: [
             {
-              isUsed: false,
-              key: "00000000000000000000000000000000000000000000000000000000",
-              path: {
-                purpose: 1852,
-                coin: 1815,
-                account: 0,
-                chain: 0,
-                index: 0,
+              hash: "tx1",
+              fees: "0",
+              timestamp: new Date(),
+              blockHeight: 1,
+              inputs: [],
+              outputs: [
+                {
+                  address: "addr",
+                  value: new BigNumber(5), // 5 lovelace
+                  tokens: [],
+                  paymentKey: "key", // match with external credential key
+                },
+              ],
+              certificate: {
+                stakeRegistrations: [],
+                stakeDeRegistrations: [],
+                stakeDelegations: [],
               },
             },
           ],
+          externalCredentials: [
+            { path: { index: 0 } as BipPath, networkId: "id", isUsed: false, key: "key" },
+          ],
           internalCredentials: [],
         } as any),
       );
+      getDelegationInfoMock.mockReturnValue(
+        Promise.resolve({
```

Comment:
Same, need to change it everywhere
```suggestion
      getDelegationInfoMock.mockResolvedValue({
```

Reply:
changed in latest commit